cityofaustin/atd-moped
# adds moped_proj_contract to activity log

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10808

## Testing
**URL to test:** 

https://10808_moped_proj_contract_activity_log--atd-moped-main.netlify.app/moped/

**Steps to test:**
- add a contract with a contractor name
- add a contract without a contractor name
- edit the other fields on the contract with a contractor name
- edit the other fields on the contract without a contractor name
- delete the contract with a contractor name
- delete the contract without a contractor name
- the activity log should reflect all six of these actions in unique ways

---
#### Ship list
